e2e/single-cluster/status_test.go

```diff
@@ -87,44 +97,46 @@ var _ = Describe("Checks status updates happen for a simple deployment", func()
 		})
 
 		It("correctly updates the status fields for GitRepos", func() {
-			out, err := k.Delete("bundle", "my-gitrepo-helm-verify", "-n", "fleet-local")
-			Expect(err).ToNot(HaveOccurred(), out)
-
-			Eventually(func() error {
-				out, err = k.Get("gitrepo", "my-gitrepo", "-n", "fleet-local", "-o", "jsonpath='{.status.summary}'")
-				if err != nil {
-					return err
-				}
-
-				expectedDesiredReady := "\"desiredReady\":0"
-				if !strings.Contains(out, expectedDesiredReady) {
-					return fmt.Errorf("expected %q not found in %q", expectedDesiredReady, out)
-				}
-
-				expectedReady := "\"ready\":0"
-				if !strings.Contains(out, expectedReady) {
-					return fmt.Errorf("expected %q not found in %q", expectedReady, out)
-				}
-
-				out, err = k.Get(
-					"gitrepo",
-					"my-gitrepo",
-					"-n",
-					"fleet-local",
-					"-o",
-					"jsonpath='{.status.display}'",
-				)
-				if err != nil {
-					return err
-				}
-
-				expectedReadyBD := "\"readyBundleDeployments\":\"0/0\""
-				if !strings.Contains(out, expectedReadyBD) {
-					return fmt.Errorf("expected %q not found in %q", expectedReadyBD, out)
-				}
-
-				return nil
-			}).ShouldNot(HaveOccurred())
+			Eventually(func(g Gomega) {
```

Comment:
I think it was because I had issues like the one described in https://github.com/aws/amazon-vpc-cni-k8s/issues/2495

I changed it to: 
```go
Eventually(func(g Gomega) {
     out, err := k.Delete("bundle", "my-gitrepo-helm-verify", "-n", "fleet-local")
     g.Expect(err).ToNot(HaveOccurred(), out)
}).Should((Succeed()))
```

and it also works. 
So that `Eventually` is no longer nested.